### ISSUE TITLE
Running the CLI with node 12 prints a deprecation message

### ISSUE THREAD

**Issue by CameronDiver** [balena-io/balena-cli#1310]
- **Cli version:** 11.2.1
- **Standalone install:** no
- **Node version:** 12.2.0
- **Npm version:** 6.9.0
- **Operating system and architecture:** arch linux x86_64

Message:
```
(node:24292) [DEP0066] DeprecationWarning: OutgoingMessage.prototype._headers is deprecated
```
---

*Please keep in mind that we try to use the issue tracker of this repository for specific bug reports & CLI feature requests. General & troubleshooting questions are encouraged to be posted to the [balena forums](https://forums.balena.io) where the community can both contribute and benefit from the answers.*

*Before submitting this issue please check that this issue is not a duplicate. If there is another issue describing the same problem or feature please add your information to the existing issue's comments.*

**Comment by srlowe** [comment]
Turns out it's the sentry logging code triggering this warning (i.e. it only occurs when sentry logs an event).

The number of occurrences may have been reduced by PR #1491, but have confirmed with dummy error that this still exists.

**Comment by srlowe** [comment]
On further investigation, this turns out to be caused by an old version of the library `timed-out` used by raven.

There is an open issue for it with sentry: https://github.com/getsentry/sentry-javascript/issues/2289

**Comment by srlowe** [comment]
Confirmed fixed by #1654 @pdcastro
